app/views/user_mailer/account_reset_request.html.erb

```diff
@@ -1,5 +1,5 @@
 <p class="lead">
-  <%= t('user_mailer.account_reset_request.intro', app: link_to(APP_NAME, AppConfig.env.mailer_domain_name, class: 'gray')) %>
+  <%== t('user_mailer.account_reset_request.intro_html', app: link_to(APP_NAME, AppConfig.env.mailer_domain_name, class: 'gray')) %>
```

Comment:
The `<%==` might not be strictly necessary combined with the new `_html` key suffix?

```suggestion
  <%= t('user_mailer.account_reset_request.intro_html', app: link_to(APP_NAME, AppConfig.env.mailer_domain_name, class: 'gray')) %>
```

https://guides.rubyonrails.org/i18n.html#using-safe-html-translations

Reply:
nice find!